.travis.yml

```diff
@@ -57,22 +57,21 @@ jobs:
 
     - name: "Code imports"
       before_install: skip
-      install:
-        - pip install isort
+      install: pip install -r requirements/dev.txt
       before_script: skip
-      script: isort --recursive --check-only --diff .
+      # Set `ensure_newline_before_comments` to true, for compatibility with
+      # black
+      script: isort --check-only --diff -n=true .
```

Comment:
This is not black, as shown in the comment?

Reply:
I meant that if I don't set `-n=true` on the isort command, it will reformat imports that have comments above them differently than black does, so in that case, running isort would undo the changes black makes